autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -39,12 +43,17 @@
 )
 from autonomy.deploy.generators.docker_compose.templates import (
     ABCI_NODE_TEMPLATE,
+    ACN_NODE_TEMPLATE,
     DOCKER_COMPOSE_TEMPLATE,
+    HARDHAT_NODE_TEMPLATE,
     TENDERMINT_CONFIG_TEMPLATE,
     TENDERMINT_NODE_TEMPLATE,
 )
 
 
+N_RESERVED_IP_ADDRESSES = 4
```

Comment:
why 4?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1423#discussion_r992342685

Reply:
This should be commented in code.